test/README.md

```diff
@@ -82,19 +83,51 @@ All FIT tests can be run from the wrapper 'run_tests.py':
       -http               forces the tests to utilize the http API protocol
       -https              forces the tests to utilize the https API protocol
       -port PORT          API port number override, default from
-                          install_config.json
+                          global_config.json
       -v V                Verbosity level of console output, default=1, Built Ins:
-                          0: No debug, 2: User script output, 4: rest calls and
-                          status info, 6: other common calls (ipmi, ssh), 9: all
-                          the rest
+                          0: No debug, 2: Display infra.run and test.run DEBUG, 4:
+                          Display test.data (rest calls and status) DEBUG, 6:
+                          infra.data DEBUG (ipmi, ssh), 9: infra.* and test.* at
+                          DEBUG_9 (max output)
+
+    Note: -v/--verbosity is being deprecated. RAC-3984 is covering that work. See
+    README.md for hints on the temporary mapping of -v to the new abilities as
+    well as a few hints on using them directly. NOTE: below is the --help output
+    from nosetests.
+    Usage: nosetests [options]
+
+    Options:
+      ... skipping output of nosetests --help down to items relevant for FIT. 'run_tests.py --help' can be used to view all available options, but untested, options ...
+
+      listing log settings:
+        --sm-list-logging   list logging settings
+        --sm-list-level=LIST_LOGGING_LEVEL
+                            detail level if --list-logging is used
+
+      setting log options:
+        note: both 'logger-name's and 'handler-name's can contain wild-card
+        characters to match multiple items.
+
+        --sm-set-logger-level=('logger-name', 'level-name-or-value')
+                            Set a logger's capture threshold. Loggers are
+                            evaluated before handlers.
+        --sm-set-handler-level=('handler-name[:logger-name]', 'level-name-or-value')
+                            Set a handler's capture threshold.
+        --sm-set-combo-level=('handler-name[:logger-name]', 'level-name-or-value')
+                            Set a handler's capture threshold AND all loggers
+                            feeding it.
+        --sm-crank-file=('file-pattern', '[handler-name[:logger-name] [level-name-or-value]]')
```

Comment:
Yeah, agreed. I had a hard time naming/describing this one. I'll try to reword and I like your name. And yeah, the "log level for handlers..." starts to feel like you are reading Dr. Suess.. ;)